src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  /* align-items: flex-start; */
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 10px;
+}
```

Comment:
do not use margin here. because if the height of the header changes, the design will be broken. use `align-items: center` and `display: flex`